TTS/tts/models/xtts.py

```diff
@@ -378,7 +379,7 @@ def get_conditioning_latents(
 
         return gpt_cond_latents, speaker_embedding
 
-    def synthesize(self, text, config, speaker_wav, language, **kwargs):
+    def synthesize(self, text, config, speaker_wav, language, speaker_id, **kwargs):
```

Comment:
I would set it to None as default. That way we don't need to update code to be compatible with this version.